skypy/galaxy/mass.py

```diff
@@ -0,0 +1,97 @@
+r"""Models of galaxy luminosities.
+
+"""
+
+import numpy as np
+
+from skypy.utils.random import schechter
+
+
+__all__ = [
+    'peng_masses',
+]
+
+
+def peng_masses(redshift, alpha, m_star, size=None,
```

Comment:
```suggestion
def stellar_mass_function(redshift, alpha, m_star, size=None,
```

This is the common Schechter stellar mass function as far as I can tell.

Reply:
Ah ok. I just wrote Peng, cause that's where I saw it the first time.